File: Pprrrrrooooojjjjeeeeecct/app/src/main/java/com/example/proecticus/ActivityToAddRashod.java
package com.example.proecticus;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.text.InputType;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;


public class ActivityToAddRashod extends AppCompatActivity {
    EditText rashodToAdd;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_to_add_rashod);
        rashodToAdd = findViewById(R.id.rashodi);
    }

    @SuppressLint("SetTextI18n")
    public void Click_AddBtn(View v)
    {
        Intent newExpense = new Intent();
        newExpense.putExtra("rashod", rashodToAdd.getText().toString());
        setResult(RESULT_OK, newExpense);
        finish();
    }
}

File: Pprrrrrooooojjjjeeeeecct/app/src/main/java/com/example/proecticus/ActivityToAddRashod.kt
package com.example.proecticus

import android.annotation.SuppressLint
import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity

class ActivityToAddRashod : AppCompatActivity() {
    var rashodToAdd: EditText? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_to_add_rashod)
        rashodToAdd = findViewById(R.id.rashodi)
    }

    @SuppressLint("SetTextI18n")
    fun Click_AddBtn(v: View?) {
        val newExpense = Intent()
        newExpense.putExtra("rashod", rashodToAdd!!.text.toString())
        setResult(Activity.RESULT_OK, newExpense)
        finish()
    }
}